SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1038,6 +1055,13 @@ void FSpatialGDKEditorToolbarModule::CloudDeploymentClicked() const
 	SpatialGDKEditorSettings->DevelopmentAuthenticationToken = DevAuthToken;
 	SpatialGDKEditorSettings->SaveConfig();
 	SpatialGDKEditorSettings->SetRuntimeDevelopmentAuthenticationToken();
+
+	// Ensure we enable bUseDevelopmentAuthenticationFlow when using cloud deployment flow.
+	SpatialGDKSettings->bUseDevelopmentAuthenticationFlow = true;
```

Comment:
should we store those fields? can't we pass in the command line args for PIE to define whether connect to local or cloud deployment? that way we could also get rid of that part of the code in the ConnectionManager that overrides the normal flow if this flag is set.

Reply:
We can't pass command line to PIE if it's running in the same process.

Reply:
ah that's annoying.. .would it be possible to instead of using this flag to check which option we have selected in the dropdown during the connection flow? I assume this also is not that straightforward due to it living in the Editor settings?

Reply:
basically: I want to get rid of having to check 2 different places for this

Reply:
I think we can use the module interface that we created for exposing the dropdown options to the engine code here as well. In a non-editor build, the module will not be present though, but checking against nullptr should suffice.

Reply:
spoke offline. will be done in a separate PR